lib/project-archiver.js

```diff
@@ -42,16 +42,32 @@ async function createArchive (config) {
     logger.warning('a file property was not found in your package.json, archiving the current directory.');
     // Get the list of files and directories
     const fileList = await helpers.listFiles(config.projectLocation);
+    // Default file exclusion rules
+    const exclusionRules = [
+      'node_modules',
+      '.git',
+      'tmp'
+    ];
+    // Parse ignore files
+    const gitignoreRules = await helpers.parseIgnoreFile(`${config.projectLocation}/.gitignore`);
+    exclusionRules.push.apply(exclusionRules, gitignoreRules);
```

Comment:
you can also merge arrays like this too:  `const merged = [...array1, ...array2];`

I wouldn't change this now, since there are other places in this file that do the same thing you are doing.  I think when i first created nodeshift, that syntax wasn't available yet.   I can add another issue for use to look at using spread to merge things

Reply:
Yeah, I used this method so I'll be consistent with the rest of the file.